dace/codegen/CMakeLists.txt

```diff
@@ -512,6 +566,64 @@ if(DACE_ENABLE_XILINX)
 
 endif() # DACE_ENABLE_XILINX
 
+# create verilator RTL simulation objects
+if(DACE_ENABLE_RTL)
+
+  # find verilator installation
+  find_package(verilator HINTS $ENV{VERILATOR_ROOT} ${VERILATOR_ROOT})
+  if (NOT verilator_FOUND)
+    message(FATAL_ERROR "Verilator was not found. Either install it, or set the VERILATOR_ROOT environment variable")
+  endif()
+
+  # check minimal version requirements
+  set(VERILATOR_MIN_VERSION "4.028")
+  if("${verilator_VERSION}" VERSION_LESS VERILATOR_MIN_VERSION)
+    message(WARNING "Please upgrade verilator to version >=${VERILATOR_MIN_VERSION}")
```

Comment:
You are right, I agree that in our case it should be an error, since otherwise DaCe will get unstable. Verilator changed  too much from version 3.xxx to version 4.xxx.